### PR TITLE
Providing more detailed information for transactions grid, data, details, and Childs

### PR DESCRIPTION
**Pull request by EvheniiaKolkhiieva**
## Purpose of this pull request

This pull request (PR) provides more detailed information for transactions grid, data, details, and transaction childs

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [ ] Other feature set, please specify:

## Affected documentation pages

- https://docs.magento.com/user-guide/sales/transactions.html?itm_source=merchdocs&itm_medium=quick_search&itm_campaign=federated_search&itm_term=transactio

## Links to Magento source code or PRs

- https://github.com/EvheniiaKolkhiieva/magento2/blob/2.4-develop/app/code/Magento/Sales/Api/TransactionRepositoryInterface.php
- https://github.com/EvheniiaKolkhiieva/magento2/blob/2.4-develop/app/code/Magento/Sales/Model/Order/Payment/Transaction.php

### CODE REVIEW

File: src/sales/transactions.md [hunk 1]
```diff
@@ -2,11 +2,81 @@
...
 On the _Admin_ sidebar, go to **Sales** > _Operations_ > **Transactions**.
 
 ![]({% link images/images/sales-transactions.png %}){: .zoom}
 _Transactions grid_
+
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|ID|A unique numeric identifier that is assigned for each transactional record.|
```
Comment (dobooth) [inline src/sales/transactions.md]: Please alphabetize things when possible: This table and the list of keys, even if it differs from the order in the admin.

**Review by dobooth — CHANGES_REQUESTED** [review]

File: src/sales/transactions.md (hunk 1 above)
Comment (EvheniiaKolkhiieva) [reply src/sales/transactions.md]: Thank you @dobooth !
Done, please approve :)

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1198`.

**Comment by leslietilling** [comment]
@dobooth  This one still needs some style edits. Let me know when it is lined up from the private repo and I will add to PR there.

**Comment by dobooth** [comment]
@leslietilling the edits can be done here and we can import again.

File: src/sales/transactions.md [hunk 2]
```diff
@@ -2,11 +2,81 @@
...
+|ID|A unique numeric identifier that is assigned to each transaction.|
+|Order ID|A unique identifier that is assigned when a customer places an order.|
+|Transaction ID|A unique numeric identifier that is assigned when a transaction happens after a customer places an order.|
+|Parent Transaction ID|The ID number of the parent transaction.|
+|Payment Method| The payment method associated with a transaction.|
+|Transaction Type|Order, Authorization, Capture, Void, Refund|
+|Closed|Whether a transaction is closed or not.|
+|Created|Time and date the transaction was created.|
+
+## To view transaction details
```
Comment (leslietilling) [inline src/sales/transactions.md:27]: We don't use infinitives in headings

```suggestion
## View transaction details
```

File: src/sales/transactions.md [hunk 3]
```diff
@@ -2,11 +2,81 @@
...
+
+1. On the _Admin_ sidebar select **Sales**.
+
+1. Click **Transaction**.
```
Comment (leslietilling) [inline src/sales/transactions.md:31]:
```suggestion
1. On the _Admin_ sidebar, go to **Sales** > **Transactions**.
```

File: src/sales/transactions.md [hunk 4]
```diff
@@ -2,11 +2,81 @@
...
+
+1. Click the entry you want to view.
+
+On the transaction details page, you can see the transaction detail and child transactions grid.
+
+### Transaction Data
```
Comment (leslietilling) [inline src/sales/transactions.md:37]:
```suggestion
### Transaction data
```

File: src/sales/transactions.md [hunk 5]
```diff
@@ -2,11 +2,81 @@
...
+
+### Child transactions
+
+Child transactions are displayed in the grid after creating invoices for [orders]{% link sales/orders.md %}. This allows you to track transaction history, following a transaction hierarchy.
+
+### Transaction Details
+
+This block includes the additional information for a given transaction. Information is displayed in the form of keys and values.
+
+**Available Key**:
```
Comment (leslietilling) [inline src/sales/transactions.md:57]:
```suggestion
This section includes the additional information for a given transaction. Information is displayed in the form of keys and values. The available keys are:
```

File: src/sales/transactions.md [hunk 6]
```diff
@@ -2,11 +2,81 @@
...
+
+1. Click **Transaction**.
+
+1. Click the entry you want to view.
+
+On the transaction details page, you can see the transaction detail and child transactions grid.
+
+### Transaction Data
+
+This block includes information regarding the transaction, and links to the order page in the **Order ID** column.
```
Comment (leslietilling) [inline src/sales/transactions.md:39]:
```suggestion
This section includes information regarding the transaction, and provides a link to the order page in the **Order ID** column.
```

File: src/sales/transactions.md [hunk 7]
```diff
@@ -2,11 +2,81 @@
...
+
+|Column|Description|
+|--- |--- |
+|Transaction ID|The transaction ID number.|
+|Parent Transaction ID|A corresponding ID number of the parent transaction. Options: Numerical ID / N/A|
+|Transaction Type|Type of transaction. Options: order, authorization, capture, void, refund.|
```
Comment (leslietilling) [inline src/sales/transactions.md:45]:
```suggestion
|Transaction Type|Type of transaction, which can be Order, Authorization, Capture, Void, or Refund.|
```

File: src/sales/transactions.md [hunk 8]
```diff
@@ -2,11 +2,81 @@
...
+On the transaction details page, you can see the transaction detail and child transactions grid.
+
+### Transaction Data
+
+This block includes information regarding the transaction, and links to the order page in the **Order ID** column.
+
+|Column|Description|
+|--- |--- |
+|Transaction ID|The transaction ID number.|
+|Parent Transaction ID|A corresponding ID number of the parent transaction. Options: Numerical ID / N/A|
```
Comment (leslietilling) [inline src/sales/transactions.md:44]:
```suggestion
|Parent Transaction ID|A corresponding ID number of the parent transaction, if applicable.|
```

File: src/sales/transactions.md [hunk 9]
```diff
@@ -2,11 +2,81 @@
...
+|Transaction Type|Type of transaction. Options: order, authorization, capture, void, refund.|
+|Is Closed|Whether a transaction is closed or not. Options: Yes/No|
```
Comment (leslietilling) [inline src/sales/transactions.md:46]:
```suggestion
|Is Closed|Whether a transaction is closed or not. |
```

File: src/sales/transactions.md [hunk 10]
```diff
@@ -2,11 +2,81 @@
...
+### Column descriptions
+
+|Column|Description|
+|--- |--- |
+|ID|A unique numeric identifier that is assigned to each transaction.|
+|Order ID|A unique identifier that is assigned when a customer places an order.|
+|Transaction ID|A unique numeric identifier that is assigned when a transaction happens after a customer places an order.|
+|Parent Transaction ID|The ID number of the parent transaction.|
+|Payment Method| The payment method associated with a transaction.|
+|Transaction Type|Order, Authorization, Capture, Void, Refund|
```
Comment (leslietilling) [inline src/sales/transactions.md:23]:
```suggestion
|Transaction Type|Type of transaction, which can be Order, Authorization, Capture, Void, or Refund.|
```

**Review by leslietilling — COMMENTED** [review]
Edits for style and consistency

**Comment by dobooth** [comment]
Opps...this one got messy. I should have deleted the imported branch. @EvheniiaKolkhiieva Please make another PR for this change, incorporating Leslies feedback.

**Comment by EvheniiaKolkhiieva** [comment]
closed in favor of https://github.com/magento/merchdocs/pull/1281

**Pull request closed without merging**